source/common/http/conn_manager_impl.cc

```diff
@@ -987,6 +987,7 @@ void ConnectionManagerImpl::ActiveStream::decodeData(
     recordLatestDataFilter(entry, state_.latest_data_decoding_filter_, decoder_filters_);
 
     state_.filter_call_state_ |= FilterCallState::DecodeData;
+    (*entry)->inline_buffered_data_ = false;
```

Comment:
Why we don't need (*entry)->inline_buffered_data_ = false; in ActiveStream::encodeData()?

Reply:
we do :) good catch